lib.php

```diff
@@ -643,3 +643,21 @@ function equella_grade_item_delete($eq) {
 function equella_get_config($configname){
     return get_config('equella', $configname);
 }
+
+/**
+ * Retrieve the userfield/username for a current user.
+ *
+ * @return string
+ */
+function mod_equella_get_userfield_value(): string {
```

Comment:
I feel these names (including the var `$equellauserfield`) are slightly confusing. Can we please tweak the names ?